cli/cobra.go

```diff
@@ -92,12 +92,8 @@ func FlagErrorFunc(cmd *cobra.Command, err error) error {
 		return nil
 	}
 
-	usage := ""
-	if cmd.HasSubCommands() {
-		usage = "\n\n" + cmd.UsageString()
-	}
 	return StatusError{
-		Status:     fmt.Sprintf("%s\nSee '%s --help'.%s", err, cmd.CommandPath(), usage),
+		Status:     fmt.Sprintf("%[2]s\n\nUsage:  %[3]s\n\nRun '%[4]s --help' for more information", cmd.Root().Name(), err, cmd.UseLine(), cmd.CommandPath()),
```

Comment:
nit: there are 2 spaces after `Usage:`

Reply:
Also, is `cmd.Root().Name()` being used?

Reply:
> nit: there are 2 spaces after `Usage:`

Yes, it looked like we're using 2 spaces for that everywhere; ISTR those may have been a `<tab>` in the past, which screwed up generating our YAML-docs, and were replaced; we should make a pass at checking where all of those are produced, and consider changing to a single one :see_no_evil:

```bash
docker --nosuch 2>&1 >/dev/null | grep Usage
Usage:  docker [OPTIONS] COMMAND
```

> Also, is `cmd.Root().Name()` being used?

Oh! Good one; I don't think it is anymore! I went through a couple of iterations; one of them was including `docker:` as prefix, but I went back-and-forth on whether that made things clearer, as it would look like (e.g.);

```
docker: Usage:  docker [OPTIONS] COMMAND
```

Which felt just _too_ verbose and somewhat confusing, so in the end I removed that.

Reply:
OH, i didn't realize we double-space in multiple areas 🙈 no problem!